### PR TITLE
clarifying words about the cf-create-org script

### PR DESCRIPTION
**Pull request by datn**

### CODE REVIEW

**Comment by afeld** [comment]
Mind if we put the usage documentation in the script itself, so it's easier to keep in sync?

File: content/docs/ops/create-org.md [hunk 1]
```diff
@@ -14,11 +14,16 @@ New sandbox organizations are automatically created when users sign in for the f
...
-1. Run the [`cf-create-org`](https://github.com/18F/cg-scripts/blob/master/cf-create-org.sh) script.
+1. Run the [`cf-create-org`](https://github.com/18F/cg-scripts/blob/master/cf-create-org.sh) script. You will need the following information, some of which is in the [form output](https://docs.google.com/spreadsheets/d/1Bdzl9n2E1MXWV4elXvZ-nYuZmmEj4PEU-u5aZlNGZF4/edit#gid=131031416)to pass to the script as arguments:
+   * **Agency**: the requesting agency, column B of the form output. 
+   * **Unique business ID**: currently provided by first-tier support.
+   * **System name**: the agency's system, column C of the form output.
+   * **Note**: currently provided by first-tier support.
+   * **Memory**: 4G by default; this should be set to a reasonable initial cap for the systems expected to live in that quota so that clients aren't shocked at their first bill. They can ask for increases as needed.
+NOTE: We don't yet have a standard convention for what constitutes a unique business ID nor what to put in the note field.
```
Comment (afeld) [inline content/docs/ops/create-org.md]: We figured that part out! From [the form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1):

> See "IAA #" in column H at https://docs.google.com/spreadsheets/d/1v4QfXGaJVy9-CZ0n6cFLHGGs_5TL1l8uCh6ZyNYjMDk/edit#gid=0

File: content/docs/ops/create-org.md [hunk 2]
```diff
@@ -14,11 +14,16 @@ New sandbox organizations are automatically created when users sign in for the f
 Non-sandbox organizations need to be created by hand. To do so:
 
 1. The [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1) should be filled out by you or the tenant.
-1. Run the [`cf-create-org`](https://github.com/18F/cg-scripts/blob/master/cf-create-org.sh) script.
+1. Run the [`cf-create-org`](https://github.com/18F/cg-scripts/blob/master/cf-create-org.sh) script. You will need the following information, some of which is in the [form output](https://docs.google.com/spreadsheets/d/1Bdzl9n2E1MXWV4elXvZ-nYuZmmEj4PEU-u5aZlNGZF4/edit#gid=131031416)to pass to the script as arguments:
```
Comment (afeld) [inline content/docs/ops/create-org.md]: Missing a space after the link.

**Comment by datn** [comment]
Thanks for the info! So, do you want me to relegate the usage info to the script? It seems pretty useful in the document about how to use the script, but I'll defer to everyone else on standard operating procedure re: docs.

File: content/docs/ops/create-org.md (hunk 1 above)
Comment (datn) [reply content/docs/ops/create-org.md]: Great! So are you saying we can use the org+system (columns E+D) to find the right line of column H and use that for the business ID? @berndverst will certainly want to know this if so.

However, I'm somewhat sure that the jury's still out on the "note" field contents.

Comment (afeld) [reply content/docs/ops/create-org.md]: > we can use the org+system (columns E+D) to find the right line of column H and use that for the business ID

👍  The thinking with linking to that doc in the form is that the onus is on the tenant (at least for teams in 18F who can access the Agreements list) to identify the right row.

> the jury's still out on the "note" field contents

Agreed! I was just following https://github.com/18F/cg-site/issues/219#issuecomment-243959351.

File: content/docs/ops/create-org.md (hunk 2 above)
Comment (datn) [reply content/docs/ops/create-org.md]: 👍

**Review by afeld — APPROVED** [review]

**Pull request merged**